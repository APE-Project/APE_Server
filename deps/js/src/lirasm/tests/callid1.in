i = immi -1;
j = immi 3;
k = immi 6;
x = immd 1.1;
y = immd 3.3;
z = immd 4.4;

res = calld callid1 cdecl i x y j k z
retd res

