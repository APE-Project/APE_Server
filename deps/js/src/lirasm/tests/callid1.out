Output is: 1.1
